data-types.md

```diff
@@ -114,6 +114,9 @@ Events represent changes in vehicle status.
 | `fuel_percent`       | Integer          | Required if Applicable | Percent fuel in vehicle, expressed between 0 and 100 |
 | `trip_ids` | UUID[] | Required if Applicable | Trip UUIDs (foreign key to /trips endpoint), required if `event_types` contains `trip_start`, `trip_end`, `trip_cancel`, `trip_enter_jurisdiction`, or `trip_leave_jurisdiction` |
 | `associated_ticket` | String | Optional | Identifier for an associated ticket inside an Agency-maintained 311 or CRM system |
+| `stop_sequence` | Integer | Required if Applicable | Order of stops for a particular trip. The values must increase along the trip, but do not need to be consecutive. |
+| `route_id` | UUID | Required if Applicable | A unique route ID for the associated route and stops. Matches [GTFS](https://gtfs.org/documentation/schedule/reference/) `route_id` |
+| `service_id` | UUID | Required if Applicable | A unique service ID for the set of dates when service is available for a route. Matches [GTFS](https://gtfs.org/documentation/schedule/reference/) `service_id` |
```

Comment:
Just making a note here for the 'Files changed' view that these fields are for **Events**.